community/udc/src/main/java/org/neo4j/ext/udc/impl/DefaultUdcInformationCollector.java

```diff
@@ -150,10 +159,39 @@ public static String filterVersionForUDC( String version )
 
         add( udcFields, FEATURES, usageData.get( UsageDataKeys.features ).asHex() );
 
+        addStoreFileSizes( udcFields );
+
         udcFields.putAll( determineSystemProperties() );
         return udcFields;
     }
 
+    /**
+     * Register store file sizes, sans transaction logs
+     */
+    private void addStoreFileSizes( Map<String, String> udcFields )
+    {
+        if (neoStoreDataSource == null) return;
+
+        try ( ResourceIterator<File> files = neoStoreDataSource.listStoreFiles( false ) )
+        {
+            addStoreFileSizes( udcFields, files );
+        }
+        catch ( IOException ignored )
+        {
+            // we just didn't report store file sizes this time around
```

Comment:
I don't think so, users would not care if their usage data wasn't sent - not their concern